packages/indexer-common/src/indexer-management/resolvers/allocations.ts

```diff
@@ -298,67 +297,68 @@ async function queryAllocations(
-        // Return early if a different protocol network is specifically requested
-        if (
-          filter.protocolNetwork &&
-          filter.protocolNetwork !== network.specification.networkIdentifier
-        ) {
-          return []
-        }
+    // Return early if a different protocol network is specifically requested
+    if (
+      filter.protocolNetwork &&
+      filter.protocolNetwork !== network.specification.networkIdentifier
+    ) {
+      return []
+    }
 
-        const {
-          networkMonitor,
-          networkSubgraph,
-          contracts,
-          specification: {
-            indexerOptions: { address },
-          },
-        } = network
-
-        const [currentEpoch, maxAllocationEpochs, epochLength] = await Promise.all([
-          networkMonitor.networkCurrentEpoch(),
-          contracts.staking.maxAllocationEpochs(),
-          contracts.epochManager.epochLength(),
-        ])
-
-        const allocation = filter.allocation
-          ? filter.allocation === 'all'
-            ? null
-            : toAddress(filter.allocation)
-          : null
-
-        const variables = {
-          indexer: toAddress(address),
-          allocation,
-          status: filter.status,
-        }
+    const {
+      networkMonitor,
+      networkSubgraph,
+      contracts,
+      specification: {
+        indexerOptions: { address },
+      },
+    } = network
+
+    const [currentEpoch, maxAllocationEpochs, epochLength] = await Promise.all([
+      networkMonitor.networkCurrentEpoch(),
+      contracts.staking.maxAllocationEpochs(),
+      contracts.epochManager.epochLength(),
+    ])
+
+    const allocation = filter.allocation
+      ? filter.allocation === 'all'
+        ? null
+        : toAddress(filter.allocation)
+      : null
+
+    const variables = {
+      indexer: toAddress(address),
+      allocation,
+      status: filter.status,
+    }
 
-        const context = {
-          currentEpoch: currentEpoch.epochNumber,
-          currentEpochStartBlock: currentEpoch.startBlockNumber,
-          currentEpochElapsedBlocks: epochElapsedBlocks(currentEpoch),
-          latestBlock: currentEpoch.latestBlock,
-          maxAllocationEpochs,
-          blocksPerEpoch: epochLength.toNumber(),
-          avgBlockTime: 13000,
-          protocolNetwork: network.specification.networkIdentifier,
-        }
+    const context = {
+      currentEpoch: currentEpoch.epochNumber,
+      currentEpochStartBlock: currentEpoch.startBlockNumber,
+      currentEpochElapsedBlocks: epochElapsedBlocks(currentEpoch),
+      latestBlock: currentEpoch.latestBlock,
+      maxAllocationEpochs,
+      blocksPerEpoch: epochLength.toNumber(),
+      avgBlockTime: 13000,
+      protocolNetwork: network.specification.networkIdentifier,
+    }
 
-        return queryAllocations(logger, networkSubgraph, variables, context)
-      },
+    const allocationsByNetwork = await queryAllocations(
```

Comment:
nit: same as above, `allocationsByNetwork` might be a confusing name? maybe just `allocationsResult`?